include/constants.h

```diff
@@ -99,13 +131,37 @@
  * This defines the internal memory that the hypervisor allocates to use
  * for allocating pages.
  *
- * Note: defined in bytes (defaults to 8MB)
+ * Note: defined in bytes (defaults to 1MB)
```

Comment:
samething... sizeof(uintptr_t), it's already tunable by changing the #define on the command line (hence why each of these have a #ifdef